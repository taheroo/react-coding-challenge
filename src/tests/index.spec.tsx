import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import Home from '../pages/index';

describe(`Home`, () => {
  it(`Should matches the snapshot`, () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
