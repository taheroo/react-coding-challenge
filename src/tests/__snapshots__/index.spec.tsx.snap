// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Should matches the snapshot 1`] = `
<div>
  <main>
    <p>
      Drop Zone
    </p>
    <div
      draggable="true"
      onDragEnter={[Function]}
    >
      <label
        htmlFor="file-input"
      >
        <div>
          <p
            className=" "
          >
            +
          </p>
          <p>
            Drop file here
          </p>
        </div>
        <input
          accept="application/pdf, doc"
          id="file-input"
          onChange={[Function]}
          type="file"
        />
      </label>
    </div>
    <p>
      File History
    </p>
    <table
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <tbody>
        <tr
          className=" "
        >
          <th>
            Filename
          </th>
          <th>
            File Size
          </th>
          <th>
            Last Modified
          </th>
          <th>
            File Format
          </th>
        </tr>
      </tbody>
    </table>
  </main>
</div>
`;
